Fix link from https://doi.org/joss.00433 to https://doi.org/10.21105/joss.00433 but the visual stay the same

yasirroni opened the pull request:
Fix link from https://doi.org/joss.00433 to https://doi.org/10.21105/joss.00433 but the visual stay the same.

<!--- Provide a general summary of your changes in the Title above -->

## Description
Fix link from https://doi.org/joss.00433 to https://doi.org/10.21105/joss.00433 but the visual stay the same.
```
[https://doi.org/joss.00433](https://doi.org/10.21105/joss.00433)
```

## Motivation and Context
Fixing the link when clicked, but not chainging the visual to make it the same as in research gate.

## How Has This Been Tested?
Just click the link.

## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [ ] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [] I have read the **CONTRIBUTING** document.
- [] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

ljvmiranda921 commented on README.md:
```diff
@@ -262,7 +262,7 @@ constrained support. The package was created with [Cookiecutter] and the
 Are you using PySwarms in your project or research? Please cite us!
 
-* Miranda L.J., (2018). PySwarms: a research toolkit for Particle Swarm Optimization in Python. *Journal of Open Source Software*, 3(21), 433, https://doi.org/joss.00433
+* Miranda L.J., (2018). PySwarms: a research toolkit for Particle Swarm Optimization in Python. *Journal of Open Source Software*, 3(21), 433, [https://doi.org/joss.00433](https://doi.org/10.21105/joss.00433)
```
You can also put 10.21105 inside the square brackets 👍

yasirroni replied:
I need to make new PR? 

*I'm not sure how to collaborate. I'll make one. Sorry, noob here.

yasirroni commented:
Fix link from https://doi.org/joss.00433 to https://doi.org/10.21105/joss.00433.

<!--- Provide a general summary of your changes in the Title above -->

## Description
Fix link from https://doi.org/joss.00433 to https://doi.org/10.21105/joss.00433 but the visual stay the same.
```
[https://doi.org/10.21105/joss.00433](https://doi.org/10.21105/joss.00433)
```

## Motivation and Context
Fixing the link when clicked to the published paper.

## How Has This Been Tested?
Just click the link.

## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [ ] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [] I have read the **CONTRIBUTING** document.
- [] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

ljvmiranda921 commented:
Awesome! Thanks a lot for your contribution, @yasirroni !
@all-contributors please add @yasirroni for docs
